internal/persistence/persistors/mysqlstore/store_category_test.go

```diff
@@ -485,17 +490,55 @@ func Test_DeleteCategory(t *testing.T) {
 			require.NoError(t, err, "unexpected error fetching the db from the tx handler")
 			require.NotNil(t, txHandlerDb, "unexpected nil tx handler db")
 
-			testCase.mutations(t, db)
-			err = m.DeleteCategory(testCtx, txHandlerDb, testCase.id)
-			testCase.assertions(t, db, testCase.id, err)
+			cat := &model.CreateCategory{
+				Name:              testCase.categoryName,
+				CategoryTypeRefId: testCase.categoryRefId,
+			}
+
+			createdCat, err := m.AddCategory(testCtx, txHandlerDb, cat)
+			testCase.assertions(t, db, createdCat, err)
 		})
 	}
 }
 
-func Test_AddCategory(t *testing.T) {
-	for _, testCase := range getAddCategoryTestCases() {
+type testCaseDeleteCategory struct {
+	name       string
+	id         int
+	assertions func(t *testing.T, db *sqlx.DB, id int, err error)
+	mutations  func(t *testing.T, db *sqlx.DB)
+}
+
+func getDeleteCategoryTestCases() []testCaseDeleteCategory {
+	return []testCaseDeleteCategory{
+		{
+			name: "success",
+			id:   1,
```

Comment:
no static values, but we talked about getting the migrated values - but better to fetch them via their name..

For example if you want to get this category "Super Admin", you can:

category, err := modelsblahblah.Categories(YOUR_FILTER_FOR_SUPER_ADMIN)

... then return that in mutation... but now this is a problem, because mutation implies you are changing something.. but whatever, let's just stick with that pattern for now.

Reply:
oh we pushed a commit where we returned the cats to their original state. basically only the store-orgs are changed here

Reply:
but thank you for the tip! we will keep that in mind moving forward